development-docs/KAFKA_VERSIONS.md

```diff
@@ -14,7 +14,7 @@ But these are some of the tasks you usually have to do.
 * If the version you are adding is the latest
   * Update the Kafka versions and protocol versions in the documentation `attributes.adoc`
   * If needed, update the examples (only in the `packaging/examples` directory):
-    * The `version` and the `inter.broker.protocol.version` fields
+    * The `version` and the `spec.kafka.metadataVersion` fields within the `Kafka` custom resource
```

Comment:
I think this is pretty understandable and I'm happy to have it merged as is ... but if you would want to be perfect, maybe you could say something like this?

```suggestion
    * The `.spec.kafka.version` and the `spec.kafka.metadataVersion` fields within the `Kafka` custom resource
    * The `.spec.version` field within the `KafkaConnect` and `KafkaMirrorMaker2` custom resources
```